app/views/credit_cards/_credit_card.html.erb

```diff
@@ -1,6 +1,14 @@
 <div class="card" style="width: 18rem">
   <div class="card-body">
-    <h5 class="card-title"><%= credit_card.name %></h5>
+    <div class="d-flex">
+      <h5 class="card-title"><%= credit_card.name %></h5>
+      <% if favorite = Favorite.find_by(user_id: current_user, credit_card_id: credit_card.id)  %>
+        <%= link_to 'Unfavorite', favorite, data: { turbo_method: :delete }, class: "btn btn-danger" %>
+      <% else %>
+        <%= render "favorites/form", favorite: Favorite.new, credit_card_id: credit_card.id %>
+      <% end %>
+      <%#= render "favorites/favorite_unfavorite", favorite: Favorite.find_by(user_id: current_user, credit_card_id: credit_card.id) %>
```

Comment:
if this logic isnt needed anymore i would recommend deleting it! helps tidy up the code